web/config/system_comment_kinds.json

```diff
@@ -0,0 +1,6 @@
+{
+  "rev_st_changed": "%author% changed review status from <b class=\"old-review-status\">{0}</b> to <b class=\"new-review-status\">{1}</b> %date%",
```

Comment:
Do we need to store the whole text, couldn't we create some more dense representation?

Reply:
We are not store the whole text in the database we just store the type of the system comment and the arguments: `rev_st_changed Unreviewed Confirmed`. We are using this file only to display the comment in the gui.